core-v2/mpi/paciente/paciente.schema.ts

```diff
@@ -97,7 +97,17 @@ export const PacienteSchema: mongoose.Schema = new mongoose.Schema({
         type: String,
         lowercase: true
     }],
-    validateAt: Date
+    validateAt: Date,
+    documentos: [{
+        fecha: Date,
+        archivos: [{
+            ext: String, id: String
+        }],
+        tipo: {
+            id: String,
+            label: String
+        }
+    }]
```

Comment:
Sugiero hacer el schema de documento separado. Y luego documentos: [documentosSchema]

Reply:
@sychus corregido

Reply:
Se creó el schema de adjuntos, pero aún falta reemplazarlo en el atributo `documentos` de `paciente.schema.ts` del core-v2
```suggestion
       documentos: [documentoAdjuntoSchema]
```
También sugiero nombrar la constante `documentoAdjuntoSchema` en mayúscula: DocumentoAdjuntoSchema

Reply:
@plammel  corregido